hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncAdmin.java

```diff
@@ -1543,9 +1544,17 @@
    * @param serverNames Server names to get slowlog responses from
    * @param logQueryFilter filter to be used if provided
    * @return Online slowlog response list. The return value wrapped by a {@link CompletableFuture}
-   */
-  CompletableFuture<List<OnlineLogRecord>> getSlowLogResponses(final Set<ServerName> serverNames,
-      final LogQueryFilter logQueryFilter);
+   * @deprecated since 2.4.0 and will be removed in 4.0.0.
+   *   Use {@link #getLogEntries(LogRequest, int)} instead.
+   */
+  default CompletableFuture<List<OnlineLogRecord>> getSlowLogResponses(
```

Comment:
See above comment

Reply:
This is deprecated API and by IA definitions, we can't change the signature of this without breaking minor release compatibility? That's why we made this default implementation to use new API internally.

Reply:
See above

Reply:
Done